src/main/scala/cromwell/caching/package.scala

```diff
@@ -1,22 +1,21 @@
-package cromwell.caching
+package cromwell
 
-import java.nio.file.{Paths, Path}
-import better.files._, Cmds._
+import better.files.Cmds._
+import better.files._
 import org.apache.commons.codec.digest.DigestUtils
 import wdl4s.values._
 
 import scala.collection.immutable.ListMap
 
 package object caching {
-  type Md5sum = String
 
-  def computeWdlValueHash(wdlValue: WdlValue): Md5sum = {
+  def computeWdlValueHash(wdlValue: WdlValue): String = {
     wdlValue match {
-      case w: WdlObject => w.value mapValues { computeWdlValueHash(_) } mkString ""
+      case w: WdlObject => w.value mapValues computeWdlValueHash mkString ""
```

Comment:
Yeah this doesn't seem right.  Furthermore I'm not even sure why this is here, this seems more appropriate to wdl4s as a method on `WdlValue`.